echopype/convert/set_groups_azfp.py

```diff
@@ -285,6 +253,16 @@ def set_vendor(self) -> xr.Dataset:
         ping_time = self.parser_obj.ping_time
         tdn = np.array(parameters["pulse_length"]) / 1e6
         channel_id = self._create_unique_channel_name(unpacked_data)
+        anc = np.array(unpacked_data["ancillary"])  # convert to np array for easy slicing
+        range_samples_xml = np.array(parameters["range_samples"])  # from xml file
```

Comment:
Since `range_samples_xml` is only used once, it can be absorbed into line 317 below.